### PR TITLE
pls check this for loop assignment

### PR DESCRIPTION
**Pull request by pingtokavita**

### CODE REVIEW

File: 06_for_Assignment01.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    console.log( index);
+        
+    }
+console.log(`2. WAP to print numbers from 50 to 40 by decrementing by 1`);
+for (let index = 50; index >=40 ; index--) {
+    console.log( index);
+        
+    }
+console.log(`3. WAP to find first 15 odd numbers`);
+for (let index = 1; index <=15; index=index+2) {
```
Comment (ketanp88) [inline 06_for_Assignment01.js:12]: Have you verified this code that this will print 15 odd numbers?

File: 06_for_Assignment01.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    console.log( index);
+        
+    }
+console.log(`4. WAP to find first 10 even numbers`);
```
Comment (ketanp88) [inline 06_for_Assignment01.js]: Same for this one. Have ypu checked weather this print 15 even numbers?

File: 06_switchCaseMonth.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    console.log(`Invalid value ${month}`);
+     break;
+
+
+  }
+}
+monthOfYear(2)
+monthOfYear(1)
+
+monthOfYear("0");
```
Comment (ketanp88) [inline 06_switchCaseMonth.js:45]: IF we pass the month as String "2" does it returns the month as February?

**Review by ketanp88 — CHANGES_REQUESTED** [review]

File: 06_for_Assignment01.js (hunk 1 above)
Comment (pingtokavita) [reply 06_for_Assignment01.js:12]: yes i have verified this code that this will print upto 15 odd numbers

**Review by ketanp88 — APPROVED** [review]

**Pull request merged**